Automattic/pocket-casts-ios
# Add import support for audio and video (movie) files

This PR builds on https://github.com/Automattic/pocket-casts-ios/pull/1035 to add support for importing audio and movie files 

## To test
1. Run the app from this branch
2. Find an audio file on your device and share it to Pocket Casts
3. ✅ The app should open on "Add file" screen
4. Save the file 
5. Tap play, the file should play. 


https://github.com/Automattic/pocket-casts-ios/assets/1335657/c323000a-6686-4700-b1ab-7ef6055971f8





## Checklist
